app/assets/javascripts/soundcloud_user.js

```diff
@@ -11,9 +11,9 @@ SoundcloudUser.prototype.searchSongs = function(){
         url: '/tracks',
         method: 'POST',
         data: {tracks: foundTracks},
-        dataType: 'JSON'
-      }).success(function(response){
-        $('body').append(response)
+      }).done(function(response){
+        $('.user-tracks').append(response);
+      }).fail(function(){
```

Comment:
![kim crying](https://cloud.githubusercontent.com/assets/50019/5631944/b6f5116a-9598-11e4-9f55-8eece5dfec5c.gif)

Dude, an empty `fail()` function.  Even `alert('bad news')` would be great.